client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.cli;
+
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestFormatUtils
+{
+    @Test
+    public void testFormatCount()
+    {
+        assertEquals(FormatUtils.formatCount(1L), "1");
```

Comment:
nit: you can import FormatUtils method statically

i know this will make it less clear that we're invking FormatUtils methods
but it will improve readability overall

Reply:
done